python/llm/example/GPU/LangChain/transformer_int4_gpu/README.md

```diff
@@ -0,0 +1,25 @@
+# Langchain examples
+
+The examples here show how to use langchain with `bigdl-llm`.
+
+## Install bigdl-llm
+Follow the instructions in [Install](https://github.com/intel-analytics/BigDL/tree/main/python/llm#install).
```

Comment:
Follow the instructions in [GPU Install Guide](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html) to install `bigdl-llm`